src/server/JasmineGraphInstanceService.cpp

```diff
@@ -295,15 +274,22 @@ void *instanceservicesession(void *dummyPt) {
             string fullFilePath =
                     utils.getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder") + "/" + fileName;
             int fileSize = atoi(size.c_str());
-            while (utils.fileExists(fullFilePath) && utils.getFileSize(fullFilePath) < fileSize) {
-                bzero(data, 301);
-                read(connFd, data, 300);
-                line = (data);
-
-                if (line.compare(JasmineGraphInstanceProtocol::FILE_RECV_CHK) == 0) {
-                    write(connFd, JasmineGraphInstanceProtocol::FILE_RECV_WAIT.c_str(),
-                          JasmineGraphInstanceProtocol::FILE_RECV_WAIT.size());
+            while (true){
+                if (utils.fileExists(fullFilePath)){
```

Comment:
This may go to and infinite loop if utils.fileExists() call evaluates to false every time. Please double check. In this case we may have to introduce small sleep to avoid busy waiting.

Reply:
Addressed